src/PlantList.tsx

```diff
@@ -307,7 +301,7 @@ function PlantList() {
                 },
             }}
             onClick={() => handleBuyRoots()}>
-            Grow Roots: {multiplier} Root for {ROOT_COST * multiplier} Sugar
+            Grow Roots: <Roots amount={multiplier} />&nbsp;for <Sugar amount={ROOT_COST * multiplier} />
```

Comment:
This line is also interesting. You can online water resources. Again, very important for consistency. If the players see the water icon they know without reading the full sentence that it's water related.